spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/actions/RewriteManifestsSparkAction.java

```diff
@@ -155,20 +155,7 @@ private RewriteManifests.Result doExecute() {
           .build();
     }
 
-    long totalSizeBytes = 0L;
-    int numEntries = 0;
```

Comment:
No longer need the number of entries. Just the size, which was moved into a helper method.